components/lwip/Kconfig

```diff
@@ -111,6 +111,24 @@ config LWIP_DHCP_DOES_ARP_CHECK
         Enabling this option performs a check (via ARP request) if the offered IP address
         is not already in use by another host on the network.
 
+config LWIP_DHCPS_LEASE_UNIT
+    int "DHCP server: Multipiler for lease time, in seconds"
+    range 1 3600
+    default 60
+    help
+        The DHCP server is calculating lease time multiplying the sent 
+        and received times by this number of seconds per unit. 
+        The default is 60, that equals one minute.
+
+config LWIP_MAX_STATION_NUM
```

Comment:
Could you please namespace this as LWIP_DHCPS_, similar to the other new config item?

Reply:
Fixed